tests/Suggestor/Suggestor.test.ts

```diff
@@ -85,16 +110,26 @@ describe.each([
         // Arrange
         const originalSettings = getSettings();
         const line = `- [ ] some task ${dueDateSymbol} to`;
-        const suggestions: SuggestInfo[] = buildSuggestions(line, 20, originalSettings);
+        const suggestions: SuggestInfo[] = buildSuggestions(line, 20, originalSettings, [] as Task[]);
         expect(suggestions[0].displayText).toContain('today');
         expect(suggestions[1].displayText).toContain('tomorrow');
     });
 
+    it('offers task suggestions for tasks too possible depend on', () => {
+        // Arrange
+        const originalSettings = getSettings();
+        const line = `- [ ] some task ${dependsOnSymbol} `;
+        const taskToDependOn = TaskBuilder.createFullyPopulatedTask();
+
+        const suggestions: SuggestInfo[] = buildSuggestions(line, line.length - 1, originalSettings, [taskToDependOn]);
+        expect(suggestions[0].displayText).toContain(taskToDependOn.descriptionWithoutTags);
+    });
+
```

Comment:
I am trying to understand this test, because I am experimenting with different ordering of the items in the suggest menu.

When I change the order, this test fails, with this for Emoji format:

```
Error: expect(received).toContain(expected) // indexOf

Expected substring: "Do exercises"
Received string:    "📅 due date"
```

And of course this for dataview format:

```
Error: expect(received).toContain(expected) // indexOf

Expected substring: "Do exercises"
Received string:    "due:: due date"
```

However, I do not understand the phrase **'offers task suggestions for tasks too possible depend on** - which means I cannot understand the test failure, nor what the actual behaviour is that's being tested.

Please could you clarify? Thanks.

Reply:
I _think_ it may be saying "when the cursor is on 'dependsOnSymbol followed by space', make sure that the first menu item is a suggested task.

Reply:
Yes it does - resolving comment.